packages/react-composites/tests/browser/call/fixture.ts

```diff
@@ -1,122 +1,32 @@
-    async ({}, use) => {
-      await startServer(path.join(__dirname, 'app'));
-      try {
-        await use(SERVER_URL);
-      } finally {
-        await stopServer();
-      }
-    },
-    { scope: 'worker' }
-  ],
-  /**
-   * Starts a test browser to load the ChatComposite example app.
-   *
-   * @returns Browser object.
-   */
-  testBrowser: [
-    // playwright forces us to use a destructuring pattern for first argument.
-    /* eslint-disable-next-line no-empty-pattern */
-    async ({}, use) => {
-      const browser = await chromium.launch({
-        // Our calling sdk streaming does not work on chromium 93, remove this line until it gets fixed
-        channel: 'chrome',
-        args: [
-          `--window-size=${Object.values(PAGE_VIEWPORT).join(',')}`,
-          '--font-render-hinting=medium', // Ensures that fonts are rendered consistently.
-          '--enable-font-antialiasing', // Ensures that fonts are rendered consistently.
-          '--disable-gpu', // Ensures that fonts are rendered consistently.
-          '--allow-file-access',
-          '--use-fake-ui-for-media-stream',
-          '--use-fake-device-for-media-stream',
-          `--use-file-for-fake-video-capture=${path.join(__dirname, '..', 'common', 'test.y4m')}`,
-          '--lang=en-US',
-          '--mute-audio'
-        ],
-        ignoreDefaultArgs: [
-          '--hide-scrollbars' // Don't hide scrollbars in headless mode.
-        ],
-        headless: true
-      });
-      try {
-        await use(browser);
-      } finally {
-        await browser.close();
-      }
-    },
-    { scope: 'worker' }
-  ],
-  /**
-   * Creates a Chat thread and adds users to the thread.
-   *
-   * @returns the created users' identities.
-   */
-  users: [
-    // playwright forces us to use a destructuring pattern for first argument.
-    /* eslint-disable-next-line no-empty-pattern */
-    async ({}, use) => {
-      const groupId = v1();
-      const users: Array<CallUserType> = [];
-      for (const displayName of PARTICIPANTS) {
-        const user = await createCallingUserAndToken();
-        user.displayName = displayName;
-        user.groupId = groupId;
-        users.push(user);
-      }
-      await use(users);
-    },
-    { scope: 'worker' }
-  ],
-  /**
-   * Loads the ChatComposite example app for each participant in a browser page.
-   *
-   * @returns Array of Page's loaded, one for each participant.
-   */
-  pages: [
-    async ({ serverUrl, testBrowser, users }, use) => {
-      const pages = await Promise.all(
-        users.map(async (user) => loadPageWithPermissionsForCalls(testBrowser, serverUrl, user))
-      );
-      await use(pages);
-    },
-    { scope: 'worker' }
-  ]
+export const test = base.extend<unknown, CallWorkerFixture>({
+  /** @returns string URL for the server. */
+  serverUrl: [createTestServer({ appDir: path.join(__dirname, 'app'), serverUrl: SERVER_URL }), { scope: 'worker' }],
```

Comment:
nitty nit: I consider paths derived from `__dirname` constants, so like to see them up top: 

    const APP_DIR = path.join(__dirname, 'app');